saa_victoria_subtheme.info.yml

```diff
@@ -1,20 +1,19 @@
 name: Victoria
 type: theme
-description: 'Stanford Alumni Association: Victoria Theme.'
+description: "Stanford Alumni Association: Victoria Theme."
 package: Stanford
-version: 8.x-1.7-dev
+version: 8.1.6
```

Comment:
This is set to the current version as it will (should) be automatically bumped to 8.1.7 on merge into 1.x